modules/ROOT/pages/apicat-about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,42 @@
+= Autocataloging APIs Using API Catalog CLI
+
+API Catalog CLI enables you to discover and catalog your API specifications, documentation files, and associated metadata as part of an automated process. You can embed the publish asset command in your automation tools, such as a CI/CD pipeline or custom scripts, to automatically trigger the cataloging of your API assets. API Catalog CLI is agnostic of CI/CD tools and runtime environments.
+
+With API Catalog CLI you can:
+
+* Identify all APIs in a project directory structure and create a <<apicat-descriptor-file,descriptor file>> to use to publish the APIs to Exchange.
+* Identify all new and changed APIs and update the descriptor file on an ongoing basis. 
+* <<apicat-trigger-cond,Conditionally trigger>> the publish based on criteria such as branches and tags.
+* <<apicat-version-strategy-cond,Conditionally set the asset version strategy>> based on criteria such as branches and tags.
+* Leverage API metadata in `exchange.json` files as part of the descriptor information if you already have those for your projects. 
+* Publish APIs described in the descriptor file by running commands at a command prompt or in a CI/CD pipeline or custom scripts.
+
+== API Catalog CLI Concepts
+
+Following are the concepts you need to know to use API Catalog CLI.
+
+[[apicat-descriptor-file]]
+=== Descriptor File
+
+A descriptor file has the information Exchange needs to catalog a set of APIs within a directory and its subdirectory structure. This file is specific to API Catalog CLI.
+
+[[apicat-exchange-json]]
+=== exchange.json
+
+The `exchange.json` file contains metadata required by Anypoint Exchange to describe published API assets. You can use this file to supplement the descriptor information provided in the API Catalog CLI descriptor file, either in-line or as an external reference.
+
+[[apicat-trigger-cond]]
+=== Trigger Conditions
+
+Trigger conditions are set in descriptor files to enable the automated publication process to determine which APIs to publish during a run of the API Catalog CLI `publish-assets` command using the trigger criteria option.
```

Comment:
```suggestion
Set trigger conditions in descriptor files to enable the automated publication process to determine which APIs to publish during a run of the API Catalog CLI `publish-assets` command using the trigger criteria option.
```

Reply:
This one is a little tough because the person creating the descriptor file may very likely NOT be the person publishing. I prefer the passive voice in this case, but we can discuss.